class.cmb-meta-box.php

```diff
@@ -295,6 +295,9 @@ static function layout_fields( array $fields ) { ?>
 				if ( isset( $field->args['repeatable_max']  ) )
 					$attrs[] = sprintf( 'data-rep-max="%s"', intval( $field->args['repeatable_max'] ) );
 
+				// Ask for confirmation before removing field.
+				if ( ! empty( $field->args['confirm_delete_field'] ) )
```

Comment:
If the user sets this to false, the data attribute won't fill in - Since we're defaulting I'd rather see a check that doesn't also apply to `false` values.